Gemfile.lock

```diff
@@ -603,7 +616,7 @@ GEM
     zeitwerk (2.6.8)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
What happened here and in the `nokogiri (1.16.2-x86_64-linux)` line? The change was introduced with the merge from upstream. But nokogiri should not be platform dependent.

Reply:
Maybe similar to https://github.com/rubygems/rubygems/issues/7304#issuecomment-1863317056 - removal of `mini_portile2` dependency may have caused the install after the merge to prefer to use the non-generic nokogiri.